.github/workflows/windows.yml

```diff
@@ -0,0 +1,131 @@
+name: Windows
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  windows:
+    name: Windows
+
+    defaults:
+      run:
+        shell: pwsh
+
+    runs-on: windows-2022
+
+    env:
+      COMPOSER_NO_INTERACTION: '1'
+      SYMFONY_DEPRECATIONS_HELPER: 'strict'
+      ANSICON: '121x90 (121x90)'
+      SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE: '1'
+
+    steps:
+      - name: Setup Git
+        run: |
+          git config --global core.autocrlf false
+          git config --global user.email ""
+          git config --global user.name "Symfony"
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Setup PHP
+        run: |
+          $env:Path = 'c:\php;' + $env:Path
+          mkdir c:\php && cd c:\php
+          iwr -outf php-7.2.5-Win32-VC15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php-7.2.5-Win32-VC15-x86.zip
+          7z x php-7.2.5-Win32-VC15-x86.zip -y >nul
+          cd ext
+          iwr -outf php_apcu-5.1.19-7.2-ts-vc15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_apcu-5.1.19-7.2-ts-vc15-x86.zip
+          7z x php_apcu-5.1.19-7.2-ts-vc15-x86.zip -y >nul
+          iwr -outf php_redis-5.3.2-7.2-ts-vc15-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_redis-5.3.2-7.2-ts-vc15-x86.zip
+          7z x php_redis-5.3.2-7.2-ts-vc15-x86.zip -y >nul
+          cd ..
+          Copy php.ini-development php.ini-min
+          "memory_limit=-1" >> php.ini-min
+          "serialize_precision=-1" >> php.ini-min
+          "max_execution_time=1200" >> php.ini-min
+          "post_max_size=2047M" >> php.ini-min
+          "upload_max_filesize=2047M" >> php.ini-min
+          "date.timezone=`"America/Los_Angeles`"" >> php.ini-min
+          "extension_dir=ext" >> php.ini-min
+          "extension=php_xsl.dll" >> php.ini-min
+          "extension=php_mbstring.dll" >> php.ini-min
+          Copy php.ini-min php.ini-max
+          "zend_extension=php_opcache.dll" >> php.ini-max
+          "opcache.enable_cli=1" >> php.ini-max
+          "extension=php_openssl.dll" >> php.ini-max
+          "extension=php_apcu.dll" >> php.ini-max
+          "extension=php_redis.dll" >> php.ini-max
+          "apc.enable_cli=1" >> php.ini-max
+          "extension=php_intl.dll" >> php.ini-max
+          "extension=php_fileinfo.dll" >> php.ini-max
+          "extension=php_pdo_sqlite.dll" >> php.ini-max
+          "extension=php_curl.dll" >> php.ini-max
+          "extension=php_sodium.dll" >> php.ini-max
+          Copy php.ini-max php.ini
+          cd D:\a\symfony\symfony
```

Comment:
I suggest using `${{ github.workspace }}` instead of hardcoding that the workspace will be in `D:\a\symfony\symfony` (which might not stay true forever)